Codecademy/gamut
# fix(Toggle): add border to dark mode circle

### Overview


Adds border to dark mode Toggles


### PR Checklist

- [x] Related to designs: https://www.figma.com/file/ReGfRNillGABAj5SlITalN/%F0%9F%93%90-Gamut?type=design&node-id=30067-40001&mode=design&t=Uhb6YQV4EmVyfz58-0
- [x] Related to JIRA ticket: [web-2250]

#### Testing Instructions


1. Go to Toggle in the SB testing env
2. Toggle to Dark Mode (using the ColorMode switcher in the top toolbar
3. Toggle should now be on light mode + dark mode

#### PR Links and Envs

The Tools Menu in Lessons is the main place this component is used in Dark Mode. 

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Mono / LE       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/3744)  | [Portal Env](http://www.google.fr/ 'Named link title')   |
